app/assets/stylesheets/application.scss

```diff
@@ -16,11 +16,23 @@
  */
 @import "foundation_and_overrides";
 
+@font-face {
+   font-family: "Oswald";
+   src: font-url('Oswald-Regular.ttf');
+}
+
+@font-face {
+   font-family: "DroidSans";
+   src: font-url('DroidSans.ttf');
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Prefer double-quoted strings